Packs/BitSight/pack_metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "BitSight",
+    "description": "BitSight Integration to get company guid, deatils, findings and to create Incidents.",
```

Comment:
I assume that 'deatil' is a type-o for 'details'?
And also the description doesn't look detailed enough.
-> Bitsight Integration will help you make more informed decisions by getting company GUIDs, details, and by fetching incidents at your desired time"